app/services/users/profile_image_generator.rb

```diff
@@ -0,0 +1,33 @@
+# rubocop:disable MixinUsage
+include CloudinaryHelper
```

Comment:
I included this here because locally it was appearing to fail when included within the module, but I might have been confused.

Reply:
Globally `include Cloudinary` does break test (and I assume it's breaking ActionView). This should be removed and custom setup needs to be added to the spec itself.

I can't find a way to get it to be included in the module but there's (generally) no need to include them here because wherever this module is being evoked from will either already include CloudinaryHelper or need to include it.  The catch is you can't call `Users::ProfileImageGenerator.call` on it's own, this module would have to be included and use the method `call` on its own. 

The other solution is to covert `ProfileImageGenerator` into a class and includes `CloudinaryHelper` within it

Reply:
I also have the feeling that globally including CloudinaryHelper there means that when Rails autoloads it will be included everywhere.